src/App.tsx

```diff
@@ -1,29 +1,48 @@
-import React from 'react';
+/* eslint-disable max-len */
+import React, { useState } from 'react';
 import './App.css';
 import { getNumbers } from './utils';
+import { Pagination } from './components/Pagination';
 
 // eslint-disable-next-line @typescript-eslint/no-unused-vars
 const items = getNumbers(1, 42).map(n => `Item ${n}`);
```

Comment:
Consider moving the `items` array inside the `App` component if it's not used elsewhere. This will ensure the array is not initialized if the component is not used.